### ISSUE TITLE
Feature request: Single header file amalgamation

### ISSUE THREAD

**Issue by monkeydom** [ToruNiina/toml11#163]
As I'd like to redistribute toml11 together with my Objective-C wrapper Library, it would be great to have a single header file amalgamation (vs the `toml.hpp` + `toml` folder) to keep the source file footprint small. Is this something this project is willing to integrate or should I do this on my end?

Or is that already an option somewhere and I'm overlooking it?

**Comment by ToruNiina** [comment]
Sorry, single-file header is currently not provided. I think just including everything should work, so I will try to provide it in the next release. At this moment, you need to generate it manually.
I think I need to automate the release process to generate the file. Automation will also prevent some mistakes as pointed out in #151 .

**Comment by monkeydom** [comment]
Yes, this kind of thing needs to be automated to also be able to quickly keep up with releases. Good to hear you are interested, so If I go ahead and build something will try to provide a pull request.

**Comment by ToruNiina** [comment]
I have just released v4.0.0 and now `single_include/toml.hpp` is supported.
